build/tailscale/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2023 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=tailscale
+PKG=ooce/network/tailscale
+VER=1.34.2
+SUMMARY="Tailscale"
+DESC="The easiest, most secure way to use WireGuard and 2FA."
+
+RUN_DEPENDS_IPS="driver/tuntap"
+
+XFORM_ARGS+="
+    -DPREFIX=${PREFIX#/}
+    -DPROG=$PROG
+    -DSERVICE=$PKG
+"
+set_arch 64
+set_gover 119
+
+build() {
+    pushd $TMPDIR/$BUILDDIR > /dev/null
+    export CGO_ENABLED=0
+    export GOOS=illumos
+    bash -x ./build_dist.sh --box ./cmd/tailscaled
```

Comment:
maybe prefix this (and other) commands with `logcmd` and add `|| logerr "<error message>"` in case it exits non-zero